tensorflow/python/estimator/estimator.py

```diff
@@ -750,6 +750,8 @@ def _verify_model_fn_args(model_fn, params):
     logging.warning('Estimator\'s model_fn (%s) includes params '
                     'argument, but params are not passed to Estimator.',
                     model_fn)
+  if tf_inspect.ismethod(model_fn):
+    _VALID_MODEL_FN_ARGS.add("self")
```

Comment:
instead of editing _VALID_MODEL..., please remove 'self' from 'args'.
also s/"/'